app/controllers/two_factor_authentication/personal_key_verification_controller.rb

```diff
@@ -6,7 +6,7 @@ class PersonalKeyVerificationController < ApplicationController
 
     def show
       analytics.track_event(
-        Analytics::MULTI_FACTOR_AUTH_ENTER_PERSONAL_KEY_VISIT, analytics_properties
+        Analytics::MULTI_FACTOR_AUTH_ENTER_PERSONAL_KEY_VISIT, context: context
```

Comment:
I might be missing something, but it looks like we're not recording `multi_factor_auth_method` at all for personal key.

Reply:
This is the `show` action, when the user visits the page that prompts them for the personal key. The method doesn't apply to the `show` action. We only need it for when they submit the form, which was already in place before this PR: https://github.com/18F/identity-idp/blob/master/app/forms/personal_key_form.rb#L28